pkg/volume/plugins_test.go

```diff
@@ -51,3 +51,71 @@ func TestSpecSourceConverters(t *testing.T) {
 		t.Errorf("Expected %v but got %v", pv.Name, converted.Name())
 	}
 }
+
+func Test_ValidatePodTemplate(t *testing.T) {
+	timeout := int64(60)
+	pod := &api.Pod{
+		ObjectMeta: api.ObjectMeta{
```

Comment:
you don't really need the ObjectMeta or most of the pod spec for these tests; you can trim down significantly here.